### ISSUE TITLE
Bug: Handle missing urls on github packages

### ISSUE THREAD

**Issue by Azraeht** [FredHappyface/.github#3]
## Bug

- [x] I have read the comment above and have completed each step
- [x] I have filled out the system info
- [x] I have described the bug, filled in the expected outcome and the actual
outcome including screenshots where appropriate

### System info

- OS: 5.15.0-79-generic #86-Ubuntu SMP Mon Jul 10 16:07:21 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
- Browser:  N/A
- Version: 2023.4.1

### Describe the bug

When licensecheck try to retrieve package info from pypi it could has an issue when JSON response from pypi content an empty `urls`  property.
An example is with https://pypi.org/pypi/model package.

In such case the code of `packageinfo.getPackageInfoPypi` will fail when trying to get the size of the package because of `size=response["urls"][-1]["size"]`

### Expected outcome

licensecheck should deal with this use case

### Actual outcome

Here is the stack trace sample:
```
Traceback (most recent call last):
  File "/home/bsantus/.cache/pypoetry/virtualenvs/sre-funky-scripts-oQ1ehjaE-py3.10/bin/licensecheck", line 8, in <module>
    sys.exit(cli())
  File "/home/bsantus/.cache/pypoetry/virtualenvs/sre-funky-scripts-oQ1ehjaE-py3.10/lib/python3.10/site-packages/licensecheck/__init__.py", line 91, in cli
    myLice, depsWithLicenses = get_deps.getDepsWithLicenses(
  File "/home/bsantus/.cache/pypoetry/virtualenvs/sre-funky-scripts-oQ1ehjaE-py3.10/lib/python3.10/site-packages/licensecheck/get_deps.py", line 189, in getDepsWithLicenses
    packages = packageinfo.getPackages(reqs)
  File "/home/bsantus/.cache/pypoetry/virtualenvs/sre-funky-scripts-oQ1ehjaE-py3.10/lib/python3.10/site-packages/licensecheck/packageinfo.py", line 114, in getPackages
    packageinfo.add(getPackageInfoPypi(requirement))
  File "/home/bsantus/.cache/pypoetry/virtualenvs/sre-funky-scripts-oQ1ehjaE-py3.10/lib/python3.10/site-packages/licensecheck/packageinfo.py", line 70, in getPackageInfoPypi
    size=int(response["urls"][-1]["size"]),
IndexError: list index out of range
```

**Comment by FredHappyface** [comment]
Thanks for the issue, and the proposed fix too

Just curious on why the issue is here and not in the repo? (I'm mainly asking as I wonder if there any action for me to improve the issue creation workflow?)

**Comment by Azraeht** [comment]
Woops!

That's because I was reading https://github.com/FredHappyface/.github/blob/master/CONTRIBUTING.md and I didn't noticed that this contributing documentation was in another repository.
Feel free to move the issue at the right place (or I can recreate it properly if you prefer)

**Comment by FredHappyface** [comment]
Nah DW about it! Had a quick look at moving it and as it's in a different org it's a pita

Thanks again for the issue and pr ☺️

I'll make a release sometime over the next week (please do feel free to poke me if it does slip my mind!)

**Comment by Azraeht** [comment]
Do you know if it could be possible to release a `2023.4.2` with this fix? I've a ongoing project which rely on your CLI and it could really great to unlock this small issue on my side :)

**Comment by FredHappyface** [comment]
Yeah no problems - released 2023.4.2 to pypi https://pypi.org/project/licensecheck/2023.4.2/
